### ISSUE TITLE
[source,xml] documentatation issue

### ISSUE THREAD

**Issue by jmcc0nn3ll** [eclipse/jetty.project#10347]
**Jetty version(s)**
Jetty 10, 11, 12

**Description**

The documentation uses a lot of [source] tags, and the ones that render XML occasionally generate a clickable link to a URL that is 404 on the eclipse jetty website—specifically the DOCTYPE section.

```
[source,xml]
<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "https://www.eclipse.org/jetty/configure_10_0.dtd">
```
This is fine.

However, with `[source,xml,subs=normal]`, the doctype link renders a clickable link to `https://eclipse.dev/jetty/configure_10_0.dtd%22%3E`.

I am opening this issue to see if there was any reason we had the source section declared like this. I can't find one and I suspect it is a long holdover to a previous way of doing things, or just a cut and paste and issue.

@sbordet, any reason we shouldn't just remove these?

**Comment by jmcc0nn3ll** [comment]
Example: https://eclipse.dev/jetty/documentation/jetty-10/operations-guide/index.html#og-xml-syntax-property

**Comment by sbordet** [comment]
Fixed by #84befe39577b3e5ab40037e89db7876de9231aa8.
